examples/slackbot/Notebook.ipynb

```diff
@@ -42,9 +34,10 @@
    "source": [
     "from datetime import datetime, timedelta\n",
     "from sklearn.model_selection import train_test_split\n",
-    "from slack_sdk.socket_mode import SocketModeClient\n",
+    "from slack_sdk.socket_mode import SocketModeClient, SocketModeResponse\n",
     "import openai\n",
     "import sparrow_pi as k\n",
+    "import sparrow_pi.windows as w\n",
```

Comment:
We should be careful about the import aliases -- it can be very confusing, especially when it's not one of the "accepted" abbreviations. For searchability, I'd suggest:

1. We use two characters (more like `pd`, `pa` which is more likely to be unique compared to a single letter). Perhaps `kt` for kaskada timestreams?
2. Instead of improting windows separately, we should do `k.windows.Since(...)` (similar to `pd.testing.assert_frame_equal(...)`).